src/Symfony/Component/Intl/Resources/bin/common.php

```diff
@@ -68,9 +68,10 @@ function get_icu_version_from_genrb($genrb)
     return $matches[1];
 }
 
-set_exception_handler(function (\Exception $exception) {
+set_exception_handler(function ($exception) {
     echo "\n";
 
+    /** @var \Exception|\Throwable $exception */
```

Comment:
From what I understand, the `\Throwable` interface is PHP7 feature (at least what I saw in the php.net) which means there would be no type defined in pre-php 7 environments. 

Also the type hint `\Exception|\Throwable` is used in other error handlers across Symfony code, so I would go with it for consistency reasons as well.